preppal-fe/src/pages/recipe.tsx

```diff
@@ -23,7 +23,7 @@ const Recipe = () => {
                     </div>
                     <div className='recipe-info'>
                         <div className='recipe-info-row'>
-                            <div className='author'> Author: {recipeInfo.author}</div>
+                            <div className='author'> Author: <Link to={"/profile/"+recipeInfo.author}>{recipeInfo.author}</Link></div>
```

Comment:
since @hwbit is changing the way recipes is using Params now, we should probably do that for authors as well